src/components/SpecialResolution/ResolutionEditor.vue

```diff
@@ -0,0 +1,318 @@
+            title="Resolution Date"
+            :nudgeRight="40"
+            :nudgeTop="85"
+            :initialValue="resolutionDateText"
+            :minDate="resolutionDateMin"
+            :maxDate="resolutionDateMax"
+            :inputRules="dateRules"
+            @emitDate="onResolutionDate($event)"
+            @emitCancel="onResolutionDate($event)"
+          />
+        </v-col>
+      </v-row>
+
+      <!-- Resolution Text -->
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4 d-none d-sm-block"
+        >
+          <label class="resolution-text-vcard-title mt-4">
+            Resolution Text
+          </label>
+        </v-col>
+        <v-col
+          cols="12"
+          sm="9"
+        >
+          <!-- For Vue 3, remove this WYSIWYG editor, consult with assets team -->
+          <tiptap-vuetify
+            id="resolutionTextEditor"
+            v-model="resolution"
+            auto-grow
+            :extensions="extensions"
+            :class="{ 'invalid': resolutionInvalid }"
+            placeholder="Full text of the resolution"
+            :card-props="{
+              flat: true,
+              style: 'background: rgba(0, 0, 0, 0.06)',
+            }"
+            :editor-properties="{ editorProps: editorProperties }"
+          />
+          <!-- Note: This component has no built in validation or rules. -->
+          <div
+            id="editorValidation"
+            :class="{'v-text-field__details': true, 'mt-2': true }"
+          >
+            <div
+              class="v-messages theme--light error--text"
+              role="alert"
+            >
+              <div class="v-messages__wrapper">
+                <v-slide-y-transition>
+                  <div
+                    v-if="resolutionInvalid"
+                    class="v-messages__message"
+                  >
+                    Resolution Text is required
+                  </div>
+                </v-slide-y-transition>
+              </div>
+            </div>
+          </div>
+          <p class="summary-text">
+            Note: If you are pasting text,
+            <strong>we recommend pasting plain text</strong>
+            to avoid formatting and font issues with PDF and printed
+            registrations. If you have pasted text other than plain text, verify
+            that your documents are correct. If they are not correct, they will
+            need to be amended.
+          </p>
+        </v-col>
+      </v-row>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'pinia-class'
+import {
+  TiptapVuetify,
+  Heading,
+  Bold,
+  Italic,
+  Strike,
+  Underline,
+  BulletList,
+  OrderedList,
+  ListItem,
+  Blockquote,
+  HardBreak,
+  HorizontalRule,
+  History,
+  Table,
+  TableCell,
+  TableHeader,
+  TableRow
+} from 'tiptap-vuetify'
```

Comment:
Could you `import * as TiptapVuetify from ...` then spread it below on line 157? Maybe not but worth asking if this could be simpler :)

Reply:
rather just leave this, so it's similar to PPR ( i think it's similar). It's going to get chucked anyways

Reply:
OK